armi/reactor/blocks.py

```diff
@@ -292,7 +292,7 @@ def getSmearDensity(self, cold=True):
 
         return smearDensity
 
-    def autoCreateSpatialGrids(self):
+    def autoCreateSpatialGrids(self, cornersUp=True):
```

Comment:
Same reservations about putting `cornersUp` switch in the generic `Block.autoCreateSpatialGrids`